java/arcs/android/devtools/DevToolsService.kt

```diff
@@ -0,0 +1,71 @@
+package arcs.android.devtools
+
+import android.app.Service
+import android.content.Intent
+import android.os.Binder
+import android.os.Bundle
+import android.os.Handler
+import android.os.HandlerThread
+import android.os.IBinder
+import android.os.Looper
+import android.os.Message
+
+class DevToolsService : Service() {
+
+    // Binder given to clients
+    private val binder = LocalBinder()
+    private lateinit var serviceLooper: Looper
+    private lateinit var serviceHandler: ServiceHandler
+    private lateinit var webSocket: DevWebSocket
+
+    /**
+     * Class used for the client Binder.
+     */
+    inner class LocalBinder : Binder() {
+        // Return this instance of LocalService so clients can call public methods
+        fun getService(): DevToolsService = this@DevToolsService
+    }
```

Comment:
May be worth defining an AIDL interface instead, to formalize things - and also to be able to support cross-process coms.